tailwind.config.mjs

```diff
@@ -13,31 +13,36 @@ const config = {
   plugins: [tailwindcssAnimate, typography],
   theme: {
     extend: {
+      keyframes: {
+        "accordion-down": {
+          from: { height: "0" },
+          to: { height: "var(--radix-accordion-content-height)" },
+        },
+        "accordion-up": {
+          from: { height: "var(--radix-accordion-content-height)" },
+          to: { height: "0" },
+        },
+      },
       animation: {
-        'accordion-down': 'accordion-down 0.2s ease-out',
-        'accordion-up': 'accordion-up 0.2s ease-out',
+        "accordion-down": "accordion-down 0.3s ease-out",
+        "accordion-up": "accordion-up 0.3s ease-out",
```

Comment:
I'm guessing these have changed because you've made changes here with the wrong auto-formatter configured.

Reply:
Confirmed fixed.